src/test/java/com/techsophy/tsf/wrapperservice/config/JWTRoleConverterTest.java

```diff
@@ -68,4 +69,19 @@ void convertTestWhileThrowingException() throws JsonProcessingException {
 
         Assertions.assertThrows(AccessDeniedException.class, () -> jwtRoleConverter.convert(jwt));
     }
+
+    @Test
+    void convertTestWhileKeyAsClientRoles() throws JsonProcessingException {
+        Jwt jwt = Mockito.mock(Jwt.class);
+        List<String> awgmentRolesList = new ArrayList<>();
+        String userResponce = "userResponse";
+        Mockito.when(webClientWrapper.webclientRequest(any(), any(), any(), any())).thenReturn(userResponce);
+        Mockito.when(objectMapper.readValue(userResponce, Map.class)).thenReturn(Map.of("key", "val"));
+        Mockito.when(objectMapper.readValue(userResponce,Map.class)).thenReturn(Map.of(CLIENT_ROLES, "val"));
```

Comment:
duplicate line?

Reply:
where is CLIENT_ROLES used in verification?

Reply:
In JWTRoleConvertor.java, Line No. 55 is 
if(userInformationMap.containsKey(CLIENT_ROLES))

It is checking if the map contains any Key as CLIENT_ROLES, To Cover this If-Block, I returned a map which contains CLIENT_ROLES as a key.

Reply:
the userResponse(fix the spelling) variable should be initialized with client roles, and real method should be called from mock